src/coreclr/vm/excep.cpp

```diff
@@ -11552,3 +11552,95 @@ MethodDesc * GetUserMethodForILStub(Thread * pThread, UINT_PTR uStubSP, MethodDe
 #endif // FEATURE_COMINTEROP
     return pUserMD;
 }
+
+
+#ifdef FEATURE_EH_FUNCLETS
+
+void SoftwareExceptionFrame::UpdateRegDisplay(const PREGDISPLAY pRD, bool updateFloats)
+{
+    LIMITED_METHOD_DAC_CONTRACT;
+
+#define CALLEE_SAVED_REGISTER(regname) pRD->pCurrentContext->regname = *dac_cast<PTR_SIZE_T>((TADDR)m_ContextPointers.regname);
+    ENUM_CALLEE_SAVED_REGISTERS();
+#undef CALLEE_SAVED_REGISTER
+
+#define CALLEE_SAVED_REGISTER(regname) pRD->pCurrentContextPointers->regname = m_ContextPointers.regname;
+    ENUM_CALLEE_SAVED_REGISTERS();
+#undef CALLEE_SAVED_REGISTER
+
+#define CALLEE_SAVED_REGISTER(regname) pRD->pCurrentContext->regname = m_Context.regname;
+    ENUM_FP_CALLEE_SAVED_REGISTERS();
+#undef CALLEE_SAVED_REGISTER
+
+    SetIP(pRD->pCurrentContext, ::GetIP(&m_Context));
+    SetSP(pRD->pCurrentContext, ::GetSP(&m_Context));
+
+    pRD->ControlPC = ::GetIP(&m_Context);
+    pRD->SP = ::GetSP(&m_Context);
+
+    pRD->IsCallerContextValid = FALSE;
+    pRD->IsCallerSPValid      = FALSE;        // Don't add usage of this field.  This is only temporary.
+}
+
+#ifndef DACCESS_COMPILE
+//
+// Init a new frame
+// funcCallDepth: the number of frames to skip when looking for the return address.
```

Comment:
Typo: funCallDepth vs. funcCallDepth

Do we need this argument at all? It is always `1`.

Reply:
This will be passed in > 1 for the follow up change that will modify FCThrow to use this new way

Reply:
You do not need to worry about FCThrows. They will be gone soon.

Reply:
cc @AaronRobinsonMSFT

Reply:
Ah, I have missed your comment on the plan to remove the FCThrow helpers. Then I can simplify it.